src/routes/app/portfolio/server/index.ts

```diff
@@ -1,10 +1,5 @@
-import { useDeleteStructure } from "./deleteStructureAction";
+export { getAvailableStructures } from './availableStructuresLoader';
+export { useCreateStructure } from "./createStructureAction";
 export { useDeleteStructure } from "./deleteStructureAction";
-import { useDeleteToken } from "./deleteTokenAction";
 export { useDeleteToken } from "./deleteTokenAction";
-import { useCreateStructure } from "./createStructureAction";
-export { useCreateStructure } from "./createStructureAction";
-import { getObservedWalletBalances } from './observerWalletBalancesLoader';
 export { getObservedWalletBalances } from './observerWalletBalancesLoader';
-import { getAvailableStructures } from './availableStructuresLoader';
-export { getAvailableStructures } from './availableStructuresLoader';
```

Comment:
Why removed all that import,export stuff? I believe it's `qwik` thing to things or sth changed in common practices? LMK please

Reply:
I wasn't fully aware of that thing in qwik. Thanks for linking the docs it was usefull and helped me to learn a purpose of those imports/exports stuff. I brought them back.